river/command.zig

```diff
@@ -62,6 +62,7 @@ const command_impls = std.ComptimeStringMap(
         .{ "keyboard-group-destroy",    @import("command/keyboard_group.zig").keyboardGroupDestroy },
         .{ "keyboard-group-remove",     @import("command/keyboard_group.zig").keyboardGroupRemove },
         .{ "keyboard-layout",           @import("command/keyboard.zig").keyboardLayout },
+        .{ "keyboard-layout-file",      @import("command/keyboard.zig").keyboardLayoutFile },
```

Comment:
I would prefer either to name this `keyboard-layout-from-file`, since that's more descriptive, or instead to add a `-from-file` option to the current `keyboard-layout` command.